ixwebsocket/IXSocketMbedTLS.cpp

```diff
@@ -39,6 +39,11 @@ namespace ix
         mbedtls_entropy_init(&_entropy);
         mbedtls_x509_crt_init(&_cacert);
         mbedtls_x509_crt_init(&_cert);
+        mbedtls_pk_init(&_pkey);
+        // FIXME: This will only work for RSA Private keys, what about other types? (ECDSA for
+        // example)
+        mbedtls_pk_setup(&_pkey, mbedtls_pk_info_from_type(MBEDTLS_PK_RSA));
+
```

Comment:
can you remove that extra newline on line 46 please ?

Reply:
Newline nemoved, apologies.